web-share/src/svelte/routes/Videos.svelte

```diff
@@ -0,0 +1,41 @@
+<!--
```

Comment:
For consistency, you may want to rename this file to `Video.svelte`.
Same would apply to `Image.svelte`

Reply:
My thinking was to have the filename match the `href`, which use `/videos` (and `/images`). `/audio` is kind of the outlier when it comes to pluralization.

Reply:
Noted.